### ISSUE TITLE
Tensorflow Version

### ISSUE THREAD

**Issue by g3mini28** [davidsandberg/facenet#989]
Am I able to use Tensorflow Version 2 (Latest release for now) for the implementation of this project?

**Comment by vdsprakash** [comment]
No, this project was not able to adapted for the TF 2.0. You can go with TF 1.13 stabilised version.
